text/0000-add-ansi-escape-codes.md

```diff
@@ -0,0 +1,63 @@
+- Feature Name: add_ansi_escape_codes
+- Start Date: 2022-02-14
+- RFC PR: (leave this empty)
+- Pony Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes the addition of more escape code functions to the ANSI primitive of the term package and the consolidation of the thirty-two existing color functions into four functions.
```

Comment:
Is this RFC adding and consolidating, or is the addition meant to result in consolidation?

Reply:
I'm going with Sean's recommendation and rewriting this RFC to only be additions and making another later for breaking changes.

Reply:
Okay! I saw that conversation and thank you for clarifying. I presume this decision means that you do not intend to remove the existing color functions in this RFC (which I was against anyway, but will voice that on the new RFC when it is ready.).